### ISSUE TITLE
Optimize PDF weight

### ISSUE THREAD

**Issue by jmaupetit** [TailorDev/pauling#45]
## Purpose

Sometimes people submit huge PDF files (>30Mb), in app rendering is quite slow in this case.

## Proposal

We may use optimized PDF files instead of the original ones. Can cloudinary optimize those PDFs?

**Comment by willdurand** [comment]
Cloudinary can upload PDFs as JPG: https://cloudinary.com/blog/uploading_converting_and_generating_thumbnails_for_pdf_documents. This could be interesting to save bandwidth, storage space, and simplify the mobile app. We would only deal with images.

**Comment by jmaupetit** [comment]
Good idea! If we always keep track of the original file URL, this would be optimal.
